src/content/posts/2022/bootstrap-5-2-1.md

```diff
@@ -0,0 +1,43 @@
+---
+author: mdo
+date: "2022-09-02T16:00:00Z"
+title: Bootstrap 5.2.1
+video:
+keywords:
+  - bootstrap
+  - release
+---
+
+Bootstrap v5.2.1 is here with fixes from our latest minor release, v5.2. As a patch release, these changes are limited to bug fixes, documentation updates, and some dependency updates.
+
+## Highlights
+
+- Added a `transparent` default hover border color CSS variable for buttons to fix a visual regression
+- Horizontal list groups with only one child now render the correct `border-radius`
+- Input groups have updated `z-index` values to ensure proper rendering of validated form fields
+- Modified the `list-group-item` selectors to better support nested imports of Bootstrap's CSS
+- Floating labels now reset their `text-align` to ensure consistent styling
+- Updated Autoprefixer to fix warnings of the `color-adjust` property (thanks to @julien-deramond on our team for reporting an issue upstream here)
```

Comment:
```suggestion
- Updated Autoprefixer to fix warnings of the `color-adjust` property (thanks to [@julien-deramond](https://github.com/julien-deramond/) on our team for reporting an issue upstream here)
```

Seems not to link automatically to profiles (see https://github.com/twbs/blog/blob/733bb890c3d130853bc866b55804a1a44bb6703c/src/content/posts/2019/bootstrap-4-3-1-and-3-4-1.md?plain=1#L20)